spec.html

```diff
@@ -28295,7 +28306,7 @@ <h1>String.prototype.charCodeAt ( _pos_ )</h1>
       <emu-clause id="sec-string.prototype.codepointat">
         <h1>String.prototype.codePointAt ( _pos_ )</h1>
         <emu-note>
-          <p>Returns a nonnegative integer Number less than 0x110000 that is the code point value of the UTF-16 encoded code point (<emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>) starting at the string element at index _pos_ within the String resulting from converting this object to a String. If there is no element at that index, the result is *undefined*. If a valid UTF-16 <emu-xref href="#surrogate-pair"></emu-xref> does not begin at _pos_, the result is the code unit at _pos_.</p>
+          <p>Returns a nonnegative integer Number less than or equal to 0x10FFFF that is the code point value of the UTF-16 encoded code point (<emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>) starting at the string element at index _pos_ within the String resulting from converting this object to a String. If there is no element at that index, the result is *undefined*. If a valid UTF-16 <emu-xref href="#surrogate-pair"></emu-xref> does not begin at _pos_, the result is the code unit at _pos_.</p>
```

Comment:
Is the “equal to 0x10FFFF” part a normative change, or a correction in the prose?

Reply:
“less than 0x110000” === “less than or equal to 0x10FFFF”, similar to how “less than 3” === “less than or equal to 2”.

It’s just easier to understand/read this way, since Unicode ranges from U+000000 to U+10FFFF. That is, `0x10FFFF` is more easily recognized than `0x110000` in this context.

Reply:
ah, duh, thanks :-) i misread it.